core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -1589,7 +1589,8 @@ abstract class RDD[T: ClassTag](
    * This is introduced as an alias for `isCheckpointed` to clarify the semantics of the
```

Comment:
oh wait I just realized this is exactly the same as `isCheckpointed`. It's kind of confusing why we have two methods that do the same thing. Is it because VertexRDD or something overrides one but not the other?